Source/dictionaries/original.json

```diff
@@ -61,6 +75,7 @@
 		"Worst" : "Vaguely Unpleasant",
 		"Wow" : "Oh GOD This is SO Boring. Please Kill Me",
 		"WOW" : "Zzzzzzzzzzz",
+        "Years" : "Minutes",
```

Comment:
It already has merge conflicts. I'll sort out tabs/spaces when I handle the conflict